src/containers/Tenant/Query/utils/getPreparedResult.ts

```diff
@@ -28,3 +28,13 @@ function escapeValue(value: string) {
         .replaceAll('\r', '\\r')
         .replaceAll('\t', '\\t');
 }
+
+export function getStringifiedQueryStats(data: unknown) {
```

Comment:
There is a similar function https://github.com/ydb-platform/ydb-embedded-ui/blob/3e463edbba920cc24650fd0c57ec279563103fc7/src/containers/Tenant/Query/ExplainResult/components/SimplifiedPlan/OperationParams.tsx#L18-L27

Let's move it to global utils